app/lib/matador/navigations.tsx

```diff
@@ -0,0 +1,27 @@
+import type { NavbarItem } from "~/lib/matador/components";
+import { DiRedis } from "react-icons/di";
+import { BiInfoCircle } from "react-icons/bi";
+import { BsArrowsAngleContract } from "react-icons/bs";
+
+const Navigations: NavbarItem[] = [
+  {
+    label: "Queues",
+    href: "./queues",
+    color: "red",
+    icon: <DiRedis size={20} />,
+  },
+  {
+    label: "Redis",
+    href: "./redis",
+    color: "blue",
+    icon: <BiInfoCircle size={20} />,
+  },
+  {
+    label: "Clients",
+    href: "./clients",
+    color: "green",
+    icon: <BsArrowsAngleContract size={20} />,
+  },
+];
+
+export default Navigations;
```

Comment:
I thought about the navigation, and I think this should be placed in `app/lib/matador/helpers/ui-helpers.ts`

Reply:
Fixed with f089a705ecee1a85e45b3512238c5af13eef525a